### DISCUSSION TITLE
Week 11 - Weekly Update - Revised Checklist

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#219] [General]
## Week 11 Progress Update
I have learned something very important about myself. I suck at design. I have a very critical eye for good design, but I can not design my own layout to save my life. I think my designs are adequate but this does need someone with an eye for design to make this actually functional. I think my layout will work for the time being, but I can already see major layout updates for the long term.

Most of my week has been hooking up the javascript to the previously static html site. At this point all the GET scripts have been created and dynamically populating all the pages. I have decided to remove one page that seem redundant which will be coming down this week. Once I have all the POST script created, I will start work on polishing up the the pages.

Here is a link to the site with the latest code:
[http://www.bksteckler.com/klokke/](http://www.bksteckler.com/klokke/)

#### Build Page Framework
 - ~~Global Header and Footers~~
 - ~~Global CSS Stylesheet~~

#### Build Homepage (HTML)
 - ~~Layout Goal Section~~
 - ~~Daily Tasks Action List~~
 - ~~Client Summary~~

#### Build Time Tracking Page
 - ~~Activity Tracker (HTML & CSS)~~
 - ~~Build Javascript for timer and data storage~~

#### Build Goal Tracker Page
 - ~~Build Goal setting layout~~
 - ~~Goal table along~~
 - Add new goal wizard

#### Build a Client Management page
 - ~~Basic layout~~
 - ~~Current Year Summary Table~~
 - ~~Current project list~~
 - Add new project / milestone / task

**Comment by CJ3315** [comment]
Just checked out the site, I think that the design of it is great for the information that you are housing. I really think that layout works the only thing is it just needs to be polished. Great work so far!

**Comment by kassyoveson** [comment]
Great work. I like the color choice and the design choices you have made. I feel that it is a very clean layout and not too cluttered. I also like the little elements you created inside of the boxes such as the box with the green squares in each corner.

**Comment by derekscholes** [comment]
Agree with the other comments, looks great!
